### ISSUE TITLE
Add AC180P support

### ISSUE THREAD

**Issue by stast1** [Patrick762/hassio-bluetti-bt#45]
Is it possible to add AC180P support?

```
Logger: homeassistant.config_entries
Source: config_entries.py:406
First occurred: 15:59:42 (1 occurrences)
Last logged: 15:59:42

Error setting up entry AC180P2341000448157 for bluetti_bt
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 406, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/bluetti_bt/__init__.py", line 55, in async_setup_entry
    coordinator = PollingCoordinator(hass, address, device_name, polling_interval, persistent_conn, polling_timeout, max_retries)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/bluetti_bt/coordinator.py", line 153, in __init__
    bluetti_device = build_device(address, device_name)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/bluetti_mqtt/bluetooth/__init__.py", line 28, in build_device
    if match[1] == 'AC200M':
       ~~~~~^^^
TypeError: 'NoneType' object is not subscriptable
```

**Comment by Patrick762** [comment]
Do you see your Powerstation as PBOX in your Bluetooth devices?

**Comment by stast1** [comment]
![image](https://github.com/Patrick762/hassio-bluetti-bt/assets/48147949/dfd0917a-6730-4c3f-9306-58cf41b14644)

**Comment by Patrick762** [comment]
I removed your picture because it contained your SN.

You could try the scan command in bluetti_mqtt. In the results we will see what protocol version it uses.

**Comment by stast1** [comment]
This is it?
[discovery_021124124813.log](https://github.com/Patrick762/hassio-bluetti-bt/files/14232071/discovery_021124124813.log)

**Comment by Patrick762** [comment]
Yes. That's the information I'm looking for. I'll need some time to check it.

I'll delete the file because this also contains personal info.

**Comment by stast1** [comment]
Super!
```
Please check which result matches your Powerstation model below
Result V1: 
Result V2:   AC180P
```

**Comment by Patrick762** [comment]
Sorry, I forgot to add the AC180P in the current beta release. I'll add it in the next (probably next friday if I have some spare time).
If you want to do it yourself, just create a new device based on a ProtocolV2Device (AC60 is similar)

**Comment by Patrick762** [comment]
I've added basic AC180P support in [0.0.19](https://github.com/Patrick762/hassio-bluetti-bt/releases/tag/0.0.19).
Can you test if the values show up?

**Comment by stast1** [comment]
Ok, I'll check now

**Comment by stast1** [comment]
Only two sensors are displayed:
AC180P Total Battery Percent
AC180P Total Power Generation

**Comment by Patrick762** [comment]
That's expected for now. Those two are the only ones available on all [v2 devices](https://github.com/Patrick762/hassio-bluetti-bt/blob/dev/custom_components/bluetti_bt/bluetti_bt_lib/base_devices/ProtocolV2Device.py)

If I have more time, I can look deeper into the discovery log

**Comment by Patrick762** [comment]
The biggest problem with v2 devices are the different amounts of inputs and outputs (f.ex. solar: 2 on EP600, 3 on EP760)

**Comment by kvanonavk** [comment]
Thank you for adding even this basic integration! It helped me to resolve a very critical issue with this device. Anyone can connect to it using Bluetti app and control it using Bluetooth when there is no any other device connected to it. I live in the apartment block and any of my neighbors could connect to my power station and mess with it. 

It seems that it's a hardware bug which can't be fixed even with the firmware update. By connecting my home assistant instance to my station using your integration I was able to block other people from connecting to it (I had to set "Persistent connection" mode for this device, hopefully it's OK). 

Now I can sleep peacefully and don't think all the time that someone can turn off my station or even worse - turn on the power lifting mode and damage some of my electronics. Thanks again for your work!
